src/typeahead/typeahead.js

```diff
@@ -499,6 +499,19 @@ angular.module('ui.bootstrap.typeahead', ['ui.bootstrap.position'])
     };
   }])
 
+  .directive('focusMe', function(){
+     return {
+       restrict: 'A',
+       link: function(scope,element,attrs){
+         scope.$watch(attrs.shouldFocus,function(newValue,oldValue){
```

Comment:
This is watching `shouldFocus`, but there is no `shouldFocus` attribute in the template - is this a mistake?